pkgs/applications/networking/mailreaders/mailpile/default.nix

```diff
@@ -1,14 +1,21 @@
-{ stdenv, fetchFromGitHub, python2Packages, gnupg1orig, openssl, git }:
+{ stdenv
+, fetchFromGitHub
+, python2Packages
```

Comment:
Can this be updated to python3?

Reply:
I'm not sure about that, mailpile 1.0 target python 3+, actually don't know where dev are, i ask here : https://github.com/mailpile/Mailpile/issues/160